openshift-knative-operator/cmd/openshift-knative-operator/kodata/ingress/latest/istio/1-net-istio-core.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
   labels:
     app.kubernetes.io/component: net-istio
     app.kubernetes.io/name: knative-serving
-    app.kubernetes.io/version: "release-v1.16"
+    app.kubernetes.io/version: "release-1.17"
```

Comment:
@creydr hmm, that doesn't seem like a pattern we want to change. Adding hold to investigate in net-* repos. 

/hold

Reply:
Updated after your fix
/unhold